ipinfo/handler.py

```diff
@@ -218,7 +218,7 @@ def getBatchDetails(
                 details = {}
                 details["ip"] = ip_address
                 details["bogon"] = True
-                return Details(details)
+                result[ip_address] = details
```

Comment:
Doesn't it need to be wrapped in `Details`, or not?

Reply:
Could be both `Details` just create instance and return them.